plugins/woocommerce-admin/client/dashboard/index.js

```diff
@@ -8,6 +8,7 @@ import { Spinner } from '@woocommerce/components';
  * Internal dependencies
  */
 import './style.scss';
+import { OrderAttributionInstallBanner } from '../order-attribution';
```

Comment:
WDYT of placing the component import before the parent component style?

Reply:
`import { OrderAttributionInstallBanner } from '~/order-attribution';` Also works and will work even if the structure changes.

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/51672/commits/749fb4dd2a5a4e34933b14b888380db0bc8cf3b3#diff-a158fec6997e1056ba460eecc464f318329f7ee6311a11da74171f4b696bb68cR10.